estate/views/estate_property_type_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+   <record id="estate_property_type_action" model="ir.actions.act_window">
+       <field name="name">Property Types</field>  <!-- This is the name of the action -->
+       <field name="res_model">estate.property.type</field> <!-- This is the model which the action applies to -->
+       <field name="view_mode">tree,kanban,form</field> <!-- are the views that will be available -->
+   </record>
+
+   <record id="estate_property_type_view_form" model="ir.ui.view">
+       <field name="name">estate.property.type.form</field>
+       <field name="model">estate.property.type</field>
+       <field name="arch" type="xml">
+           <form string="Estate Property Type">
+               <sheet>
+                   <div class="oe_title">
+                       <h1 class="mb16">
+                           <field name="name"/>
+                       </h1>
+                   </div>
+               </sheet>
+           </form>
+       </field>
+   </record>
+  
+   <record id="estate_property_type_view_tree" model="ir.ui.view">
+       <field name="name">estate.property.type.tree</field>
+       <field name="model">estate.property.type</field>
+       <field name="arch" type="xml">
+           <tree>
+               <field name="name"/>
+           </tree>
+       </field>
+   </record>
+
+   <record id="view_estate_property_type_search" model="ir.ui.view">
+       <field name="name">estate.property.type.search</field>
+       <field name="model">estate.property.type</field>
+       <field name="arch" type="xml">
+           <search string="Estate Property">
+               <field name="name" string="Name" />
```

Comment:
I think `string="Name"` it's not necessary, can you please check it functionally?